step-templates/proxmox-deploy-lxc.json

```diff
@@ -171,16 +181,6 @@
 			"DisplaySettings": {
 				"Octopus.ControlType": "Checkbox"
 			}
-		},
-		{
-			"Id": "55f7937b-4a5f-49bb-b129-f589c3ce51cc",
-			"Name": "Proxmox.LXC.Password",
-			"Label": "LXC Root Password",
-			"HelpText": "This will be the root password once the LXC container has been created",
-			"DefaultValue": "",
-			"DisplaySettings": {
-				"Octopus.ControlType": "Sensitive"
-			}
 		}
 	],
 	"StepPackageId": "Octopus.Script",

```

Comment:
The `ExportedAt` should ideally be updated to indicate a change has also occurred :)